Testscripts/Windows/VERIFY-INFINIBAND-MultiVM.ps1

```diff
@@ -79,6 +79,12 @@ function Main {
 			return "SKIPPED"
 		}
 
+		# IBM Platform MPI shows 32-bit binary complexity in Ubuntu
```

Comment:
Platform MPI is the only one that needs PKEY to be specified explicitly. If you are not running Platform MPI, there is no need to query PKEY and specify it.

Reply:
I think it's still a good idea to verify the PKEYs are there and they are what we expect

Reply:
sure, sounds good.

Reply:
ok, then I keep the pkey checking modules for all.